docs/pages/guides/building-for-tv.mdx

```diff
@@ -84,15 +84,17 @@ At this time, TV applications work with the following libraries and APIs listed
 - [LinearGradient](/versions/latest/sdk/linear-gradient/)
 - [Localization](/versions/latest/sdk/localization/)
 - [Manifests](/versions/latest/sdk/manifests/)
+- [MediaLibrary](/versions/latest/sdk/media-library/)
 - [NetInfo](/versions/latest/sdk/netinfo/)
 - [Reanimated](/versions/latest/sdk/reanimated/)
 - [SafeAreaContext](/versions/latest/sdk/safe-area-context/)
+- [SecureStore](/versions/latest/sdk/securestore/)
 - [SplashScreen](/versions/latest/sdk/splash-screen/)
 - [Svg](/versions/latest/sdk/svg/)
 - [Updates](/versions/latest/sdk/updates/)
 - [Video](/versions/latest/sdk/video/)
```

Comment:
[Expo Tracking Transparency](https://docs.expo.dev/versions/latest/sdk/tracking-transparency/) can also be added here. However, the question remains whether it makes sense to display the tvOS compatible libraries redudantly in the docs 🤔

Reply:
[Expo System UI](https://github.com/expo/expo/blob/main/packages/expo-system-ui/ios/ExpoSystemUI.podspec) is missing as well.

Reply:
[Expo Network](https://docs.expo.dev/versions/latest/sdk/network/) as well.

Reply:
Done.